packages/arb-token-bridge-ui/src/util/index.ts

```diff
@@ -26,3 +26,16 @@ export function preloadImages(imageSources: string[]) {
     image.src = imageSrc
   })
 }
+
+export const loadEnvironmentVariableWithFallback = ({
```

Comment:
Using `??` to determine fallback of env vars doesn't work since it accepts an empty string

Reply:
the idea was not to have env vars undefined in any case, will improve the function in the next PR to throw if undefined